appveyor.yml

```diff
@@ -0,0 +1,22 @@
+version: 1.3.{build}-{branch}
+
+image: Visual Studio 2015
+
+cache:
+    - C:\MumbleBuild\cache -> appveyor.yml
+
+clone_folder: C:\MumbleBuild\mumble
+
+environment:
+    MUMBLE_BUILDSCRIPT: 1.3.x/mumble-win64-static.cmd
+    MUMBLE_ENVIRONMENT_VERSION: win64-static-no-ltcg-1.3.x-2017-02-02-ec94ddb-790
```

Comment:
That requires a different cache invalidation strategy. We could download a marker file in the init stage that we can use for invalidation I think. Also not having build env version checked in will mean any patch will always build with the newest one instead of the one current for the branch. Maybe we want that but it makes hard to know whether a CI result is still valid. It definitely prevents the builds there from being reproducible.

An alternative suggestion would be to have somthing like transifexbot for new environment versions. What would be neat about that is that the appveyor would automatically test building mumble with the changed environment before we can ever merge it. Not sure how valuable that is in practice.